gui/CMakeLists.txt

```diff
@@ -29,9 +29,10 @@ if(webgui)
 
    add_subdirectory(webgui6)
 
+   add_subdirectory(browsable)
+   add_subdirectory(browserv7)
```

Comment:
I'm confused: browserv7 is always enabled, even if root7 is not? That looks weird...

Reply:
`browserv7` is just subfolder name, it is `RBrowser` functionality

Reply:
I can change directory name to `rbrowser`